AvgStarArray.jsx

```diff
@@ -0,0 +1,71 @@
+// LIBRARY IMPORTS
+import React from 'react';
+import styled from 'styled-components';
+
+// LOCAL IMPORTS
+import average from '../../utilities/helpers.js';
+import EmptyStar from './EmptyStar.jsx';
+import QuarterStar from './QuarterStar.jsx';
+import HalfStar from './HalfStar.jsx';
+import ThreeQuarterStar from './ThreeQuarterStar.jsx';
+import FullStar from './FullStar.jsx';
+
+// STYLED COMPONENT
+const StyledAvgStarArray = styled.div`
+  display: flex;
+  flex-direction: row
+`;
+
+const AvgStarArray = function CreateAvgStarArray() {
+  // TEMPORARY DATA
+  const avgRating = 3.5;
```

Comment:
To make this dynamic, this line can be updated with the following steps:
1. import ProdContext
2. useContext to get meta data (where ratings data lives, and what is required to use average function)
3. set avgRating equal to a call to average(meta.ratings). 

I think that should make everything after this line work!